src/form.py

```diff
@@ -95,165 +75,19 @@
         NOTE_FREQS[key], SAMPLE_RATE, MAX_AMPLITUDE, DURATION
     ).generate_wave()
 
-selected = sine_waves
-
+# fmt: off
```

Comment:
I'm a little hesitant to wrap code in `# fmt: off` / `# fmt: on`. Mainly because Katie mentioned not disabling rules for the assignments.

Unfortunately it seems like black is truly opinionated without a way to customize line wrapping. We might want to write another comment above explaining why we are turning off formatting just for this section.